sdk/textanalytics/azure-ai-textanalytics/src/test/java/com/azure/ai/textanalytics/TextAnalyticsAsyncClientTest.java

```diff
@@ -243,6 +243,25 @@ public void detectLanguageNoneCountryHint(HttpClient httpClient, TextAnalyticsSe
                 .verifyComplete());
     }
 
+//    /**
```

Comment:
nit: i'd remove this commented out code, you can't really reach too many documents with detect_language, since if you go over, you will actually run into a batch size overload error instead

Reply:
removed this test